docs/plus/cronjob.md

```diff
@@ -0,0 +1,93 @@
+# CronJob
+
+CronJob resource is responsible for creating recurring [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/). The job recurrence is determined by a [Schedule]() provided. You can also specify a custom schedule by passing a [cron pattern](https://en.wikipedia.org/wiki/Cron) to Schedule's cron method.
+
+CronJob is similar to a job but it is suitable when there is a need to run a job indefinitely following a schedule. These repitive jobs can be utilized for recurring tasks like backing up a database, pinging a server for health checks, creating snapshots of system and much more.
+
+Some of the properties that can be configured for a CronJob are,
+
+* ### `concurrencyPolicy`
+  Policy to determine how to handle concurrent executions of jobs. The valid values for this policy are, Allow, Forbid and Replace.
+
+* ### `startingDeadline`
+  Deadline for a job to run by if it misses its scheduled time. 
+  
+  For instance, if a job was scheduled to run every 5 minutes and you have mentioned a *startingDeadline* of 30 seconds. Then the job status would be updated as failed if the job does not start by 5 minutes and 30 seconds after the start time of previous job.
+
+* ### `successfulJobsRetained`
+  Number of successful job runs retained during CronJob execution.
+
+* ### `failedJobsRetained`
+  Number of failed job runs retained during CronJob execution.
+
+## Creating a CronJob
+
+You can create a cronjob using either a cdk8s+ CronJob construct or cdk8s+ Job construct's schedule method. 
+
+The following are examples for each.
+
+### Creating a CronJob using CronJob construct:
+
+```typescript
+import { App, Chart, Duration, Schedule } from 'cdk8s';
+import * as kplus from 'cdk8s-plus-24';
+
+const app = new App();
+const chart = new Chart(app, 'Chart');
+
+// lets define database backup job properties which we can pass to our CronJob resource
+const dbBackupJobProps: k8splus.JobProps = {
+  activeDeadline: Duration.minutes(5),
+  backoffLimit: 5,
+  containers: [
+    {
+      image: 'databack/mysql-backup',
+    }
+  ]
+};
+
+// lets setup a cron job here which backups mysql database.
+const backupCronJob = new k8splus.CronJob(this, 'BackupCronJob', {
+  jobProperties: dbBackupJobProps,
+  schedule: Schedule.rate(Duration.minutes(5)),
+  timezone: k8splus.Timezone.PDT,
+  concurrencyPolicy: k8splus.CronJobConcurrencyPolicy.ALLOW,
+  startingDeadline: Duration.seconds(30),
+  suspend: false,
+  successfulJobsRetained: 5,
+  failedJobsRetained: 5,
```

Comment:
Don't include any non required properties. This should be the very most basic example.
Then other properties should be described in their own section along with an appropriate use case.

Reply:
Ahh, ok. Will update it.